pkg/service/protocolDataService/withdrawals.go

```diff
@@ -41,6 +42,7 @@ func (pds *ProtocolDataService) ListQueuedWithdrawals(
 		)
 		where
 		    qsw.block_number <= @blockHeight
+			AND (csw.withdrawal_root IS NULL OR csw.block_number <= @blockHeight)
```

Comment:
sorry, this should be `csw.block_number > @blockHeight`. an uncompleted withdrawal at blockHeight is one where there is no completed event or the block of completion was after blockHeight